docs/_issues/2024-09-13-issue-7.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Snapp Mobile iOS Newsletter 7
+issue: 7
+tags: [Generics, VoiceOver in SwiftUI, SwiftData Expressions]
+date: 2024-09-13
+---
+
+## Swift
+
+### [Mastering Generics in Swift: From Basics to Advanced Techniques](https://swiftshorts.com/2024/09/10/mastering-generics-in-swift-from-basics-to-advanced-techniques/)
+
+Generics are one of the most powerful features of Swift, and much of the Swift standard library is built with generic code. Understanding the topic is the key to elevate your command of the Swift programming language as it enables you to write flexible, reusable functions and types that can work with any type, subject to requirements that you define.
+
+### [Using Custom Components in Swift's Regex](https://dandylyons.github.io/posts/using-custom-components-in-swifts-regex/)
+
+Do you knew that you can create your own custom parsers through the `CustomConsumingRegexComponent` protocol? This article explains how and comes with an example of creating a custom phone number component.
```

Comment:
Sorry if I suggesting to fix a correct `Do you knew`. I believe it might be the author style.
```suggestion
Did you know that you can create your own custom parsers through the `CustomConsumingRegexComponent` protocol? This article explains how and comes with an example of creating a custom phone number component.
```

Reply:
Fixed with [this commit](https://github.com/Snapp-Mobile/ios-newsletter/pull/8/commits/99ae97a52ffa1ddc72f8fcb92cba77b4f7f7f851)